pack/nodejs/helm.ts

```diff
@@ -61,14 +67,12 @@ export namespace v2 {
                 const chart = path.quotePath(nodepath.join(chartDir.name, config.chart));
                 const release = shell.quote([releaseName]);
                 const values = path.quotePath(overrides.name);
+                const defaultValues = path.quotePath(nodepath.join(chartDir.name, config.chart, "values.yaml"));
```

Comment:
Is this file guaranteed to exist? What happens if it doesn't?

Reply:
It's guaranteed to exist or it's not a valid chart.